src/config/dynamic-theme-fixes.config

```diff
@@ -13172,6 +13172,31 @@ td {
 
 ================================
 
+mafreebox.freebox.fr
+
+CSS
+* {
+filter:none !important;
+}
+.desktop-icon-outer {
+    color: #fff;
+}
+body {
+    background-image: linear-gradient(rgb(14 14 14 / 50%), rgb(40 40 40 / 50%)),url(../resources/images/fbx/bg_freeboxos.svg);
```

Comment:
I believe an absolute URL would be more reliable

Reply:
Strangely, the full path doesn't work.
The default value on the website is already a relative path.